.env.development

```diff
@@ -1,4 +1,4 @@
 PORT=4500
-REACT_APP_API_URL=http://127.0.0.1:3000
+REACT_APP_API_URL=http://192.168.31.230:3000
```

Comment:
Another change that is related to your local environment, is `yarn.lock`. Currently we use npm due to some legacy stuff in Docker... We'll switch to yarn soon. But currently package-lock.json is present. We shouldn't have yarn.lock as it will cause confusion.